composer.json

```diff
@@ -35,7 +36,6 @@
         "laminas/laminas-zendframework-bridge": "^1.0",
         "mezzio/mezzio": "^3.0",
         "mezzio/mezzio-router": "^3.0",
-        "ocramius/package-versions": "^1.3",
```

Comment:
Uhm, this "just works" when using `^1.0` - not sure we need to change it at all?

Reply:
Travis has composer v2 installed by default.
So there is either rolling back travis to use composer v1 or switch over to the deprecated package. 🤷🏼‍♂️

Reply:
@Ocramius Replacement of the package was necessary, because of some incompatibility issues (see [my comment](https://github.com/mezzio/mezzio-tooling/pull/7#issuecomment-730001746)). If there is a more suitable solution, I do not know it, Feel free to share. 😊

@boesing Not sure if I follow correctly. Composer 2.0 supports 5.3+.